blob/driver/driver.go

```diff
@@ -22,6 +22,9 @@ import (
 	"time"
 )
 
+// ReaderOptions controls Reader behaviors.
```

Comment:
Same advice here: consider leaving a comment that it is empty for future compatibility.